src/pages/Content/getCommitElements.ts

```diff
@@ -0,0 +1,54 @@
+import ElementUtil from "@pages/Content/utils/element";
+
+type CommitInfo = { title: string; link: string; hash: string };
+
+export default function getCommitElements() {
+  const prTimelineTitles: HTMLElement[] = getPRTimelineTitles();
+  const prCommitInfos: CommitInfo[] = prTimelineTitles.map(getCommitInfo);
+  const commitElements: HTMLDivElement[] =
+    prCommitInfos.map(createCommitElement);
+
+  return commitElements;
+}
+
+function createCommitElement(commentInfo: CommitInfo): HTMLDivElement {
+  const commitElement = document.createElement("div");
+  const spanElement = document.createElement("span");
+  spanElement.textContent = commentInfo.title;
+
+  const anchorElement = document.createElement("a");
+  anchorElement.text = commentInfo.hash.slice(0, 7);
+  anchorElement.href = commentInfo.link;
+
+  commitElement.append(spanElement);
+  commitElement.append(anchorElement);
+
+  return commitElement;
+}
+
+function getCommitInfo(element: Element): CommitInfo {
+  const commitTitle = element.textContent;
+  const commitLink = "https://github.com" + element.getAttribute("href");
+  const commitHash = commitLink.split("/").reverse()[0];
+
+  const commit: CommitInfo = {
+    title: commitTitle || "",
+    link: commitLink,
+    hash: commitHash,
+  };
+
+  return commit;
+}
+
+function getPRTimelineTitles(): HTMLElement[] {
+  const prTimeLines = ElementUtil.arrayFrom(
+    document.querySelectorAll('[data-test-selector="pr-timeline-commits-list"]')
+  );
```

Comment:
다음처럼 직접 사용하는게 어떨까요? 유틸함수를 만든 이점이 어떤게 있는지 잘모르겠는데 설명부탁드립니다~!
```suggestion
  const prTimeLines = Array.from(
    document.querySelectorAll('[data-test-selector="pr-timeline-commits-list"]')
  );
```

Reply:
유틸함수로 얻으려고 했던건 타입 캐스팅이랑 ArrayFrom의 polyfill, 그리고 추후에 Element를 다루는 다른 유틸과 모아서 관리하려는 목적이 있었습니당

생각해보니 타입 캐스팅은 유틸함수에서 함부로 해주는게 좀 위험할 것 같고, polyfill은 어차피 크롬 확장이라 의미가 없네요;; 유틸을 모아서 처리하려던건 지금 단계에선 불필요한 생각인 것 같아서 지워줬어용

https://github.com/parkhojeong/commits-for-github/pull/3/commits/bca7970ee9a1cf558bf6840fa9ea95ff4d4e466a